code/modules/flufftext/Hallucination.dm

```diff
@@ -605,27 +599,26 @@ Gunshots/explosions/opening doors/less rare audio (done)
 /obj/effect/fake_attacker/New(loc, mob/living/carbon/T)
 	..()
 	my_target = T
-	spawn(300)
-		qdel(src)
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, src), 300)
 	step_away(src,my_target,2)
-	spawn(0)
-		attack_loop()
```

Comment:
Never use addtimer for spawn(0), use INVOKE_ASYNC(src, .proc/attack_loop) instead